src/ray/core_worker/transport/dependency_resolver.cc

```diff
@@ -64,8 +64,8 @@ void LocalDependencyResolver::CancelDependencyResolution(const TaskID &task_id)
 
 void LocalDependencyResolver::ResolveDependencies(
     TaskSpecification &task, std::function<void(Status)> on_dependencies_resolved) {
-  std::unordered_set<ObjectID> local_dependency_ids;
-  std::unordered_set<ActorID> actor_dependency_ids;
+  absl::flat_hash_set<ObjectID> local_dependency_ids;
```

Comment:
Swiss table is preferred unless pointer stability is needed, which is not the case here.
Quite a few illustrations, for example https://abseil.io/blog/20180927-swisstables